### DISCUSSION TITLE
blog/rails-8-whats-coming

### DISCUSSION THREAD

**Discussion by giscus** [harrison-broadbent/railsnotes-comments#9] [RailsNotes Comments]
# blog/rails-8-whats-coming

Work on Rails 8 is starting, so I dug through the official GitHub milestone to break down all the upcoming goodies for you. There's plenty coming, including Solid Queue, Solid Cache and Kamal becoming defaults, better support for PWAs, an official LSP, and more!

https://railsnotes.xyz/blog/rails-8-whats-coming

**Comment by rafaeldev** [comment]
I'm excited for Propshaft! It's comming!

**Comment by defifund** [comment]
Very good guide, very useful, thank you very much.
